calico-cloud/networking/configuring/node-local-dns-cache.mdx

```diff
@@ -0,0 +1,60 @@
+---
+description: Install NodeLocal DNSCache
+---
+
+# Use NodeLocal DNSCache in your cluster
+
+## Big picture
+
+Set up NodeLocal DNSCache to improve DNS lookup latency.
+
+## Before you begin
+
+### Required
+
+Follow these [steps](https://kubernetes.io/docs/tasks/administer-cluster/nodelocaldns/) to enable NodeLocal DNSCache connectivity.
+
+### Unsupported
```

Comment:
I would mention GKE has a known issue running NodeLocalDNS Cache on GKE while upgrading to Calico Cloud if NodeLocalDNS was enabled _before_ upgrading. It works if you enable this feature if you upgrade to cloud first. https://tigera.atlassian.net/browse/EV-3878.

Reply:
@Glen-Tigera Good point. However, we do not track bugs/known issues for CC. I've asked PM if we need to start putting in RN, or just tracking internally. For now, let's not add this bug to the doc.